src/runtime/components/elements/Accordion.vue

```diff
@@ -3,7 +3,7 @@
     <HDisclosure v-for="(item, index) in items" v-slot="{ open, close }" :key="index" :default-open="defaultOpen || item.defaultOpen">
       <HDisclosureButton as="template" :disabled="item.disabled">
         <slot :item="item" :index="index" :open="open" :close="close">
-          <UButton v-bind="{ ...omit(ui.default, ['openIcon', 'closeIcon']), ...$attrs, ...omit(item, ['slot', 'disabled', 'content', 'defaultOpen']) }" class="w-full">
+          <UButton v-bind="{ ...ui.default.button, ...$attrs, ...omit(item, ['slot', 'disabled', 'content', 'defaultOpen']) }">
```

Comment:
Why remove the `omit` here?

Reply:
I have moved all options related to the `button` into a new key in the `config`, so there is no need to `omit` ?

Reply:
This doesn't follow the real pattern as you can pass a `color` or `variant` to `UAccordion` directly trough the `v-bind="$attrs"`.

Reply:
I guss that !

Reply:
You can add the `class` directly inside the `ui.default` I guess.